src/hotspot/share/runtime/thread.hpp

```diff
@@ -311,15 +311,9 @@ class Thread: public ThreadShadow {
   volatile uint32_t _suspend_flags;
 
  private:
-  int _num_nested_signal;
-
   DEBUG_ONLY(bool _suspendible_thread;)
 
  public:
```

Comment:
In a product build we will have:

private:
public:

is it possible the compiler may warn about this?

Reply:
I don't believe so.